assign_rights/assemble.py

```diff
@@ -17,9 +23,21 @@ def retrieve_rights(self, rights_ids):
             rights_shells.append(RightsShell.objects.get(pk=ident))
         return rights_shells
 
-    def calculate_dates(self, end_date):
-        """docstring for calculate_dates"""
-    pass
+    def calculate_dates(self, object, grouping_end_date):
```

Comment:
"Grouping" here refers to the content described in the POST request; I'm wondering if this variable name should be changed to avoid confusion.